.gitignore

```diff
@@ -72,5 +72,6 @@ coverage.xml
 # Front-End #
 #############
 node_modules/
+lib/
```

Comment:
I love that idea. It gives us the niceness of not checking `lib` into source control and we don't need to maintain the .npmignore file